src/fragments/lib/geo/js/search.mdx

```diff
@@ -0,0 +1,304 @@
+```
+
+### Search for suggestions
+
+The `Geo.searchForSuggestions()` API enables you to search for suggestions by free-form text, such as a place, address, city, or region.
+
+```javascript
+import { Geo } from "@aws-amplify/geo";
+
+Geo.searchForSuggestions("Amazon Go Store")
+```
+
+Similar to `Geo.searchByText()` API, customize your search results further by providing:
+- `countries` - to limit the search results to given countries (specified in [ISO Alpha-3 country codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3))
+- `maxResults` - to limit the maximum result set
+- `biasPosition` - to act as the search origination location
+- `searchAreaConstraints` - to limit the area to search inside of
+- `searchIndexName` - to use a different Location Service search index resource than the default
+
+ **Note:** Providing both `biasPosition` and `searchAreaConstraints` parameters simultaneously returns an error.
+
+```javascript
+const searchOptionsWithBiasPosition = {
+  countries: string[], // Alpha-3 country codes
+  maxResults: number, // 50 is the max and the default
+  biasPosition: [
+    longitude // number
+    latitude // number,
+  ], // Coordinates point to act as the center of the search
+  searchIndexName: string, // the string name of the search index
+}
+
+const searchOptionsWithSearchAreaConstraints = {
+  countries: ["USA"], // Alpha-3 country codes
+  maxResults: 25, // 50 is the max and the default
+  searchAreaConstraints: [SWLongitude, SWLatitude, NELongitude, NELatitude], // Bounding box to search inside of
+  searchIndexName: string, // the string name of the search index
+}
+
+Geo.searchForSuggestions('Amazon Go', searchOptionsWithBiasPosition)
+```
+This returns a list of suggestions (places and their respective `placeId` if available) that match the search constraints.
+
+```javascript
+// returns
+[
+  {
+    text: "Amazon Go, 2131 7th Ave, Seattle, WA, 98121, USA",
+    placeId: "8fd9d4c6-2527-4190-a7df-0dae352c9dc6"
+  },
+  {
+    text: "Amazon Go, 1906 Terry Ave, Seattle, WA, 98101, USA",
+    placeId: "5d04d071-dea2-4d86-bfce-86bd6a8f4787"
+  }
+]
+```
+
+In cases where `placeId` is not available on the list of suggestions as below, use ``searchByText`` to search for the selected place by text.
+
+```javascript
+Geo.searchForSuggestions("Amazon", { MaxResults: 5 })
+
+// returns
+[
+  {
+    text: "Amazon Go",
+  },
+  {
+    text: "Amazon 4-star",
+  }
+]
+
+Geo.searchByText('Amazon Go', { MaxResults: 5 })
+```
+
+This returns places and their coordinates that match the search text.
+
+
+### Search by PlaceId
+
+The `Geo.searchByPlaceId()` API enables you to search for a place by a `placeId`, which is a unique opaque token for a place returned by the provider.
+
+<Callout>
+
+**Note:** Upgrade to the latest Amplify CLI version (greater than version 10.4) and do an ``amplify push`` to update your IAM policies to allow for ``getPlace`` API.
```

Comment:
Nit: While this is still accurate, I'm wondering if this is confusing since CLI version has to be 12.5.1 or above to use other code samples

Reply:
Good catch missed updating cli version here, updated